pom.xml

```diff
@@ -204,7 +204,7 @@
     <jboss.logging.processor.version>2.2.1.Final</jboss.logging.processor.version>
     <jboss.logging.version>3.5.3.Final</jboss.logging.version>
     <jetty-assembly-descriptors.version>1.1</jetty-assembly-descriptors.version>
-    <jetty-quiche-native.version>0.20.0</jetty-quiche-native.version>
+    <jetty-quiche-native.version>0.20.1</jetty-quiche-native.version>
```

Comment:
Did you release 0.20.1 to maven central yet?

https://repo1.maven.org/maven2/org/mortbay/jetty/quiche/jetty-quiche-native/

Reply:
Yes, I have. I suppose it will appear on maven central in a few minutes.

Reply:
0.20.1 is now on central and the build passed.